crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -1943,9 +1943,14 @@ impl ExecutionComponents {
         metrics: Arc<ResolverMetrics>,
     ) -> Self {
         let native_functions = sui_framework::natives::all_natives();
+        let enable_move_vm_paranoid_checks = false;
```

Comment:
Why setting them to false if we already have the config?

Reply:
I did not see how to access `ExpensiveSafetyCheckConfig` here (it is not inside `protocol_config`)

Reply:
Right, you would need to pass it from the constructor of per epoch store

Reply:
I will do that.